package.json

```diff
@@ -10,7 +10,7 @@
     "watch": "npm run clean && webpack --progress --colors --watch --mode=\"development\"",
     "clean": "rm -rf dist/",
     "lint": "eslint \"**/*.ts\"",
-    "test": "npm run lint; jest"
+    "test": "npm run lint && jest --onlyChanged --watch"
```

Comment:
I think we should have a separate script for watch, as `npm test` should be a script that ends (this will keep running, watching for changes). Maybe `test-watch` to be aligned with other scripts in this project?

Reply:
:+1: